fineract-doc/src/docs/en/chapters/demos/video-demonstration.adoc

```diff
@@ -0,0 +1,3 @@
+= VIDEO DEMONSTRATION
+
+Apache Fineract / Mifos X Demo (November 2016) - <https://www.youtube.com/watch?v=h61g9TptMBo>
```

Comment:
This is really old.  Let's delete or create a new one.  I think  it should be a demo of the APIs

Reply:
@jdailey Done.